modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/src/main/resources/META-INF/resources/js/pages/apps/edit/utils.es.js

```diff
@@ -102,3 +102,8 @@ export function validateSelectedFormViews(formViews = []) {
 
 	return {duplicatedFields, errorIndexes};
 }
+
+export const filterDataLayoutList = (appWorkflowDataLayoutLinks) =>
+	appWorkflowDataLayoutLinks.filter(
+		({dataLayoutId}) => dataLayoutId !== undefined
+	);
```

Comment:
Hey @gabrielpramos , I think that we don't need to create this util for filter undefined dataLayouts you can just call the filter on all occurrences, and you can use `dataLayoutId` instead of `dataLayoutId !== undefined` on implementation.